### PR TITLE
Issue 282: Added distance value in store card

### LINKED ISSUE #282: Show distance value in the store cards
Feature request from @spdongre: "Stores nearby should have facility to mention the radius 1 Km, 3 Km, 5 Km, 8 km, 10 km etc."

### PR DESCRIPTION
**Pull request by xsihe**
Fixes #282 
I wasn't sure where to put the distance (or if imperial units were needed), so I just left it as it is below the store name for now.

### CODE REVIEW

File: src/Homepage/ResultBlock.js [hunk 1]
```diff
@@ -81,6 +82,11 @@ function ResultBlock(props) {
               textToHighlight={result.name}
             />
           </h5>
+          <div className="mt-n1">
+            <small className="text-muted d-inline-block mt-2">
+              <strong>Within {distanceKM} km</strong>
```
Comment (arjun27) [inline src/Homepage/ResultBlock.js:87]: can we use translations for this string, so that it can be localized in the different languages?

Comment (xsihe) [reply src/Homepage/ResultBlock.js:87]: Sure, I'll work on it

Comment (Siddharth11) [inline src/Homepage/ResultBlock.js:87]: Why do we need that extra wrapping div with negative margin 🤔
This markup generates the same layout

```suggestion
            </h5>
            <small className="text-muted d-block mt-1">
              <strong>Within {distanceKM} km</strong>
            </small>  
```

Comment (xsihe) [reply src/Homepage/ResultBlock.js:87]: Good catch, I changed it to `mt-1,` but it was originally in `<div>` to be on a separate block.

**Comment by xsihe** [comment]
Since I don't know Hindi or Portuguese, I left those pages with just x km for distance string, so maybe someone else can change the strings later?

**Comment by Siddharth11** [comment]
> Since I don't know Hindi or Portuguese, I left those pages with just x km for distance string, so maybe someone else can change the strings later?

@xsihe Yes, we can add it later. Thanks for fixing 👍

**Comment by asanjeevrao** [comment]
Hey @xsihe, good work on this. Noticed a few bugs that we'll need to fix - 

1) when users search for a location using the search bar, the distances of stores are being calculated from the searched pin's location. Distances from stores should always be shown from current location.
Eg: when I search for 'Bangalore' but my current location is Vancouver
image
 
2) it seems wrong that we're showing default distances using user IP location and not geo-locations as the current location. Here's what I see on page load - 
image
vs what I see after using locate me - 
image

One solution could be to ask location permissions on page load and then decide use geo-location or IP location depending on whether or not user gives permissions 

what do you think @arjun27 @Siddharth11 

3) Small one but we could remove the 'within' since the accuracy is always 0.1Km (pretty small anyway). The distance can them move to the same line as the Store Name. Should help us keep store cards more compact. 

We should roll back this feature till we fix it 1, ideally 2 as well. 3 is a good to have and should not be a release blocker. @Siddharth11

**Comment by Siddharth11** [comment]
@asanjeevrao @xsihe i haven't rolled back all the changes. Just hid it behind a feature flag -
https://github.com/covid-maps/covid-maps/commit/ac52f093819ae0270425120af39b0cd60014cb11. Use won't see this anymore

**Comment by xsihe** [comment]
@asanjeevrao Sure, no problem, for 3, I only added 'within' in front of distance just in case if some users would mistake radius distance for actual travel road distance. 

Also, would you open issue(s) for 1 and 2?

**Comment by asanjeevrao** [comment]
@xsihe created #298 & #299

**Pull request merged**